tools/pre-commit

```diff
@@ -86,6 +91,34 @@ then
     test $jshint_result -ne 0 && jshint_information
     exit 1
   fi
+
+  if [ -r "$L10N_XFAIL_LIST" ]; then
+    echo "mozL10n check:"
+    # warn about files being xfailed
+    for file in $js_changed_files ; do
+      one_was_ignored=""
+      if grep -q $file "$L10N_XFAIL_LIST" ; then
+        one_was_ignored="1"
+        echo "$file: mozL10n errors are ignored. Please consider fixing errors and removing it from $L10N_XFAIL_LIST."
+      fi
+    done
+
+    test -n "$one_was_ignored" && mozL10n_information
+
+    to_lint=`echo "$js_changed_files" | sort | comm -13 "$L10N_XFAIL_LIST" - | tr '[:space:]' ' '`
+    # check for deprecated mozL10n API uses
+    for file in $to_lint ; do
+      one_failed=""
+      if grep -q "mozL10n.get" "$file" ; then
```

Comment:
This is easily escapable in the code too; maybe file a separate bug to make it work in eslint and add the bug number in a comment here ?